src/lib/data/projects.ts

```diff
@@ -167,3 +169,38 @@ export async function getGoverningEntityIDsForProjects({
 
   return result;
 }
+
+export const getConditionFieldsForProjects = async ({
+  database,
+  projects,
+}: {
+  database: Database;
+  projects: Map<ProjectId, ProjectData>;
```

Comment:
What do you think about making this method easier to use outside your use-case, by requiring less project data?
```suggestion
  projects: Map<ProjectId, Pick<ProjectData, 'projectVersionPlan'>>;
```